CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+2023.08.25
+  - Use yyyy.mm.dd version scheme, instead of yyyy.mm.release
+  - Added oneAPI support to CMake
+  - Fixed int64 support
+  - More robust Makefile configure doesn't require CUDA or ROCm to be in
+    compiler search paths (CPATH, LIBRARY_PATH, etc.)
+  - Added `gemqrt` to multiply by Q from QR
```

Comment:
Should we mention that its specifically the `Q` from `geqrt`?

Reply:
It's in the `gemqrt` documentation; the Changelog doesn't go into high detail.